src/main/java/com/microsoft/sqlserver/jdbc/SQLServerDriver.java

```diff
@@ -67,7 +67,8 @@ enum SqlAuthentication {
     ActiveDirectoryIntegrated,
     ActiveDirectoryMSI,
```

Comment:
rename this to ActiveDirectoryManagedIdentity since that is the current feature name